cmd/update/config.go

```diff
@@ -0,0 +1,24 @@
+package update
+
+import (
+	"github.com/lyft/flytestdlib/config"
+)
+
+//go:generate pflags Config
+
+var (
+	defaultConfig = &Config{
+	}
+	section       = config.MustRegisterSection("update", defaultConfig)
+)
+
+// Config hold configuration for project update flags.
+type Config struct {
+	ActivateProject bool `json:"activateProject" pflag:",Activates the project specified as argument."`
```

Comment:
Can you also add a small section to the contributing guide, that all `flags` are camel cased

Reply:
Done